components/list/list-item-mixin.js

```diff
@@ -384,7 +384,8 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin-bottom: -4px;
 				margin-top: -3px;
 			}
-			:host([draggable][selectable]) [slot="add"] {
+			:host([draggable][selectable][_hovering]) [slot="add"],
+			:host([draggable][selectable][_focusing]) [slot="add"] {
 				padding-inline-end: 6px;
 			}
```

Comment:
My concern was that if hovering/focusing on a sibling then the line extended a bit too far right. However, this is actually consistent with current behaviour.